client/src/components/Form/Form.jsx

```diff
@@ -4,13 +4,18 @@ import useStyles from './styles';
 import { Box, Button, TextField, Typography } from '@material-ui/core';
 import { MuiPickersUtilsProvider, KeyboardDatePicker } from '@material-ui/pickers';
 import DateFnsUtils from '@date-io/date-fns';
+import Alert from '@material-ui/lab/Alert';
+
 
 const Form = () => {
   const styles = useStyles();
   const history = useHistory();
   const [firstName, setFirstName] = useState(null);
   const [lastName, setLastName] = useState(null);
   const [selectedDate, handleDateChange] = useState(null);
+  const [phoneNumber, setPhoneNumber] = useState(null);
```

Comment:
Why is `phoneNumber` being added if it's not being used? Should we hold off on setting this state variable until it actually has functionality?

Reply:
I forgot to remove that was also working on the other fields. Good catch.

Removed it.